### ISSUE TITLE
On Symfony 3.3.2, problem parsing routing/cart.yml

### ISSUE THREAD

**Issue by wadjeroudi** [Sylius/Sylius#8165]
```
[Symfony\Component\Config\Exception\FileLoaderLoadException]                                                                                                              
  The file "alias: sylius.order                                                                                                                                             
  path: '" does not exist in /var/www/vendor/sylius/sylius/src/Sylius/Bundle/AdminApiBundle/Resources/config/routing/cart.yml (which is being imported from "/var/www/vend  
  or/sylius/sylius/src/Sylius/Bundle/AdminApiBundle/Resources/config/routing/main.yml"). 
```
Am I the only one getting this error ?
If I remove this part, everything works.
```
sylius_admin_api_carts:
    resource: |
        alias: sylius.order
        path: '/'
        only: [index]
        grid: sylius_admin_api_cart
        section: admin_api
        serialization_version: $version
    type: sylius.resource_api
```

**Comment by tchapi** [comment]
I'm having this error too. It's only the `path` bit that poses a problem, but I cannot isolate the reason why on my side. It seems that this `path` parameter chokes when it's empty, or blank, or a single slash.

It works flawlessly if you put an arbitrary string in it, like `path: 'cart'` for instance, but I don't know if it breaks parts of the API or not.

**Comment by wadjeroudi** [comment]
I think the path should be omitted and it will use /  by default? We are in a prefix case. (/carts)

**Comment by tchapi** [comment]
That must be the case, but I'm not sure though.

**Comment by jarekkowol** [comment]
I have the same problem.

**Comment by pjedrzejewski** [comment]
Sylius does not support Symfony 3.3 yet, although we plan to release a small Beta3 (without BC breaks) and then switch to Symfony 3.3 and PHP 7.1 only. According to #8114.

**Comment by wadjeroudi** [comment]
@pjedrzejewski in #8114 you spoke about PHP 7.1, and SF 4. You already fixed 3.3 issues in master, it is the last one (known), could you please fix it on the master before closing ?

**Comment by lchrusciel** [comment]
Sylius will reach stable before Symfony4, so we will stay with SF 3.3. SF 4 was just mentioned as an example. Anyway, I think it should stay reopen(it is still a bug).
